### DISCUSSION TITLE
Migration from OneDrive to Azure

### DISCUSSION THREAD

**Discussion by WiggyToo** [ep1cman/unifi-protect-backup#166] [General]
Just felt like posting to say thank you to the creator of this tool (@ep1cman) - it's been running flawlessly backing up to OneDrive until I ran out of space, (1TB) so I ended up migrating it to Azure Files Storage.

The process was pretty simple in the end, I stopped the service running (it runs in an Container on OpenMediaVault), and used a tool (GoodSync) to copy all the backed up files from OneDrive to Azure, this took a while as there were 18,000 plus files that had accumulated since early July when I started.

Once all the files were moved and I'd added the AZURE remote to the rclone.conf (connect to AZURE using key and connection string), updated the Compose file to change from the old rclone destination to the new, restarted everything and off it went, added the missing events that had happened while the service was stopped and then carried on.

This process did make me think of another use case, which would be more to archive old video rather than back up all, for example the last 6 months could be on the Unifi Cloud Key (or other Protect device) and the 6 months before that on other storage using Rclone - this would help with cases where Protect storage runs out.

Regards and thanks for all the work,
Gav

**Comment by ep1cman** [comment]
Hi,

Thanks for your message, I'm glad to heard things are working well for you!
